test/test_common_theurgy.rb

```diff
@@ -0,0 +1,130 @@
+require 'minitest/autorun'
+load 'test/test_harness.rb'
+
+include Harness
+
+class TestPerceiveHealth < Minitest::Test
```

Comment:
```suggestion
class TestCommonTheurgy < Minitest::Test
```

Reply:
Curious, this suggestion is marked resolved but the test file still shows as `TestPerceiveHealth`. Is the commit to perform the rename done locally and yet to be pushed? Thanks